src/Schema/Domain/Content/Worker/FieldDefinition/AddFieldValueToDomainContent.php

```diff
@@ -37,10 +43,16 @@ public function work(Builder $schema, array $args)
 
     private function getDefinition(FieldDefinition $fieldDefinition)
     {
-        return [
+        $definition = [
             'type' => $this->fieldDefinitionMapper->mapToFieldValueType($fieldDefinition),
             'resolve' => $this->fieldDefinitionMapper->mapToFieldValueResolver($fieldDefinition),
         ];
+
+        if (isset($this->argsMappers[$fieldDefinition->fieldTypeIdentifier])) {
+            $definition['argsBuilder'] = $this->argsMappers[$fieldDefinition->fieldTypeIdentifier]->mapToFieldValueArgs($fieldDefinition);
+        }
```

Comment:
If you like to merge `FieldDefinitionArgsMapper` interface into `FieldDefinitionMapper` in 2.0 maybe we could go with the following approach in 1.0: 

```php
        if ($this->fieldDefinitionMapper instanceof FieldDefinitionArgsMapper) {
            $definition['argsBuilder'] = $this->fieldDefinitionMapper->mapToFieldValueArgs($fieldDefinition);
        }
```

* `ezplatform_graphql.field_definition_args_mapper` tag and compiler pass will be not needed
* It will force users to implement `FieldDefinitionArgsMapper` together with `FieldDefinitionMapper`. When  `mapToFieldValueArgs ` will added to `FieldDefinitionMapper` in most cases will be enought to remove `implements FieldDefinitionArgsMapper` during the migration.

Reply:
Note that there are third party packages implementing that feature already (matrix, page...), and we need to keep BC on it.